dataframe/selection.xml

```diff
@@ -0,0 +1,11 @@
+<lcgdict>
+    <selection>
+
+            <function name="pt" />
```

Comment:
I don't know anything about this stuff, but shouldn't `cos_theta` be also here?

Reply:
True, to use  these lambdas from the ROOT interpreter that'll be necessary. Will add it.

Reply:
Also forgot the namespace here that I added only later.